src/aircloak/JsonApi/AircloakConnection.cs

```diff
@@ -26,8 +35,17 @@ public AircloakConnection(JsonApiClient apiClient, string dataSourceName, TimeSp
             this.cancellationTokenSource = new CancellationTokenSource();
         }
 
+        /// <summary>
+        /// Gets a value indicating whether a cancellation was requested for this connection.
+        /// </summary>
```

Comment:
FYI. I think you can just use `<inheritdoc/>` in cases like this to use the docstring from the base class or interface. 
```suggestion
        /// <inheritdoc/>
```